std/array.d

```diff
@@ -3173,7 +3174,51 @@ if (isDynamicArray!A)
         }
     }
 
+    /**
+     * Gives a string in the form of `Appender!(A)(data)`.
+     *
+     * Params:
+     *     w = A `char` accepting
+     *     $(REF_ALTTEXT output range, isOutputRange, std, range, primitives).
+     *     fmt = A $(REF FormatSpec, std, format) which controls how the array
+     *     is formatted.
+     * Returns:
+     *     A `string` if `writer` is not set; `void` otherwise.
+     */
+    string toString()() const
+    {
+        // Even though the Appender might represent a string,
+        // make a copy of the data anyway because we don't want
+        // manipulations of the toString result to change the
+        // data of the Appender
+        auto app = appender!string();
+        auto spec = singleSpec("%s");
+        // different reserve lengths because each element in a
+        // non-string-like array uses two extra characters for `, `
+        static if (isSomeString!A)
```

Comment:
FWIW we wanted to deprecate this trait...

https://github.com/dlang/phobos/pull/5284, https://github.com/dlang/phobos/pull/5137

Reply:
`isSomeString` is far too ubiquitous to deprecate IMO. Johnathan already modified it to not recognize `enum`s, which I think was the main issue.